lib/flutter_calendar_carousel.dart

```diff
@@ -8,55 +8,11 @@ import 'package:flutter_calendar_carousel/classes/event_list.dart';
 import 'package:intl/date_symbol_data_local.dart';
 import 'package:intl/intl.dart' show DateFormat;
 
+import 'src/src.dart';
```

Comment:
Wouldn't it be better to import each `default` styles here to be more clear?

Reply:
Also, I think the naming of this `src.dart` should be changed to something different. Why not just import this directly from `default_styles.dart`?

Reply:
I was thinking of putting all the files on `src.dart` that way the import section would be clean. But I guess it won't make much of a difference, importing directly or importing from src. I'll change to import directly so it is more clear.